common/convert/convert.go

```diff
@@ -107,3 +107,8 @@ func SplitFloatDecimals(input float64) (baseNum, decimalNum int64, err error) {
 	}
 	return baseNum, decimalNum, nil
 }
+
+func BoolPtr(condition bool) *bool {
+	b := condition
```

Comment:
I'm suddenly curious about the purpose of these bool funcs. I know, after I told you to change them :D
So.. Is the point of it to have a singular version of `true` and `false` declared so we only ever reference one bool and no need to create another?

Reply:
I did originally want to have FalsePtr & TruePtr but there was no good package to place these in that don't cause import cycles 

if i put it in common i can't use it in logger because common imports logger
utils is imported by engine and a few other packages

I didn't want to create a new package for two lines under common so convert and a func that returns the type seemed like the best option to keep it somewhat clean

There was discussion earlier about a common exported const package that has common words and other things we use such as "ok" "failure" etc that does not import any package and is just variables unsure if this will go ahead or not